lib/tasks/redirect.js

```diff
@@ -24,6 +26,21 @@ class TaskRedirect extends Task {
       } catch (err) {
         this.logger.info(err, `Task:performAction error redirecting to ${this.actionHook}`);
       }
+    } else if (cs.application.requestor._isAbsoluteUrl(this.actionHook)) {
+      const baseUrl =  this.cs.application.requestor.baseUrl;
+      const newUrl = URL.parse(this.actionHook);
+      const newBaseUrl = newUrl.protocol + '//' + newUrl.host;
+      if (baseUrl != newBaseUrl) {
```

Comment:
just to be sure, you have tested where an absolute url is provided but has the same base url as the current, and seen that it does not update the base url?

Reply:
yes